### ISSUE TITLE
KeyError with `make test` and Python 3.5.3 and 1.6b1 release

### ISSUE THREAD

**Issue by jfbu** [sphinx-doc/sphinx#3660]
#### Procedure to reproduce the problem
```
make test
```

#### Error logs / results
```
Checking dependencies...
Temporary files will be placed in /Users/johndoe/xxx/sphinx/tests/build.
Running Sphinx test suite (with Python 3.5.3)...
Traceback (most recent call last):
  File "/opt/miniconda3/lib/python3.5/site-packages/_pytest/config.py", line 325, in _getconftestmodules
    return self._path2confmods[path]
KeyError: local('/Users/johndoe/xxx/sphinx/tests')

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/opt/miniconda3/lib/python3.5/site-packages/_pytest/config.py", line 356, in _importconftest
    return self._conftestpath2mod[conftestpath]
KeyError: local('/Users/johndoe/xxx/sphinx/tests/conftest.py')

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/opt/miniconda3/lib/python3.5/site-packages/_pytest/config.py", line 362, in _importconftest
    mod = conftestpath.pyimport()
  File "/opt/miniconda3/lib/python3.5/site-packages/py/_path/local.py", line 662, in pyimport
    __import__(modname)
  File "/opt/miniconda3/lib/python3.5/site-packages/_pytest/assertion/rewrite.py", line 216, in load_module
    py.builtin.exec_(co, mod.__dict__)
  File "/Users/johndoe/xxx/sphinx/tests/conftest.py", line 11, in <module>
    import util
  File "/Users/johndoe/xxx/sphinx/tests/util.py", line 25, in <module>
    from sphinx import application
  File "/Users/johndoe/xxx/sphinx/sphinx/application.py", line 33, in <module>
    from sphinx.environment import BuildEnvironment
  File "/Users/johndoe/xxx/sphinx/sphinx/environment/__init__.py", line 44, in <module>
    from sphinx.util.websupport import is_commentable
  File "/Users/johndoe/xxx/sphinx/sphinx/util/websupport.py", line 10, in <module>
    from sphinxcontrib.websupport.utils import is_commentable  # NOQA
ImportError: No module named 'sphinxcontrib'
ERROR: could not load /Users/johndoe/xxx/sphinx/tests/conftest.py

make: *** [Makefile:91: test] Error 4
```

### Environment info
- OS: Mac 10.9.5
- Python version: 3.5.3
- Sphinx version: 1.6-release branch at 1.6b1 tag

**Comment by jfbu** [comment]
Actually, this is fixed by

   pip install sphinxcontrib-websupport

but I am not one hundred percent sure I issued the `make test` in the correct repertory initially. (I am a bit in confused state right now, anyway now `make test` completed successfully at my locale).

Relates: #3254, #3654

**Comment by tk0miya** [comment]
Yes, `sphinxcontrib-websupport` is added to our dependency list.
In #3654, it is added to `test-reqs.txt`, so we need to install it manually before testing.
It seems it is resolved by yourself, right?

**Comment by jfbu** [comment]
Yes, problem is resolved but I need to `pip install sphinxcontrib-websupport` not only for testing, without it `make html` complains. But the message is clear, although it mentions `sphinxcontrib.websupport` not `sphinxcontrib-websupport` which user needs to learn about from `pip search`.

**Comment by tk0miya** [comment]
`sphinxcontrib-websupport` is already on dependency list. So it will be installed on installing Sphinx-1.6b1.
https://github.com/sphinx-doc/sphinx/blob/1.6-release/setup.py#L54

**Comment by tk0miya** [comment]
I think this is already resolved. please close this if agreed.

**Comment by jfbu** [comment]
On 1.6b2 it was not yet solved, I just tried a virtualenv and ``pip install sphinx==1.6b2``, then 
```
(testsphinx) $ sphinx-build -h
Error: The 'sphinxcontrib' module cannot be found. Did you install Sphinx and its dependencies correctly?
```
I am now closing.

**Comment by jfbu** [comment]
Update: this got fixed at 0135948
